readme.md

```diff
@@ -10,7 +10,7 @@ Lighthouse is integrated directly into the Chrome Developer Tools, under the "Au
 
 **Run it**: open Chrome DevTools, select the Audits panel, and hit "Run audits".
 
-<img width="500px" alt="Lighthouse integration in Chrome DevTools" src="https://user-images.githubusercontent.com/2301202/40556658-4ef7d128-6002-11e8-903e-5224894a7cca.png">
+<img src="https://github.com/GoogleChrome/lighthouse/raw/master/assets/example_dev_tools.png" alt="Lighthouse integration in Chrome DevTools" width="500px">
```

Comment:
Is there a reason we're moving to `raw/master` instead of hash-based?

raw/master can just be super confusing once master gets ahead of older versions. We don't update the images that often, think we could stick with hashes?

Reply:
hm true

https://www.npmjs.com/package/@code-therapy/packages-with-readme-relative-images#conclusion lays out how annoying this is right now

Reply:
Hmm I definitely find the hashed links confusing because I never know where they are coming from.  And I think it would be much easier to keep them updated if they were in the repo/easy to find and update.

There is the raw.github link for these images e.g. `https://raw.githubusercontent.com/GoogleChrome/lighthouse/master/assets/lighthouse-logo.png` but I guess that comes right back to the raw/master issue, this is just the resolution after replacing `blob` with `raw`.

Not sure exactly how to handle this then.

Reply:
Seems like in the example @paulirish sent over he uses the form `raw.githubusercontent.com/.../HEAD/.../...png` so it removes the master hardcoding.  Seems like what he was doing might be good for us to maintain absolute paths, but still have some flexibility.